benchmarks/models_config/sam_fast_torch_compile_gpu.yaml

```diff
@@ -0,0 +1,18 @@
+---
+sam_fast_vit_b:
+    process_batch_size_4:
+        benchmark_engine: "ab"
+        url: https://torchserve.pytorch.org/mar_files/sam-fast-vit-b-process-batch-size-4.tar.gz
```

Comment:
So, we don't need to have any model weights locally on the machine?

Reply:
The model weights are still required locally on this machine. However, I have packaged the `custom_handler.py` and `model-config.yaml` files with `torch-model-archiver` and uploaded the artifact to s3. 

The model weights are present at this path in the machine: `/home/ubuntu/segment_anything_fast/sam_vit_b_01ec64.pth`

The `model-config.yaml` file for this artifact is:
```
responseTimeout: 300
minWorkers: 4
maxWorkers: 4
handler:
    profile: true
    model_type: "vit_b"
    sam_checkpoint: "/home/ubuntu/segment_anything_fast/sam_vit_b_01ec64.pth"
    process_batch_size: 4

```

Reply:
sounds good. Thanks